cmd/remote_client.go

```diff
@@ -264,19 +265,45 @@ func (cli *Client) RemoteLogin(c *clipkg.Context) error {
 
 // Withdraw will withdraw LINK to an address authorized by the node
 func (cli *Client) Withdraw(c *clipkg.Context) error {
-	if len(c.Args()) < 2 {
+	args := c.Args()
+	if len(args) < 2 {
 		return cli.errorOut(errors.New("withdrawal requires an address and amount"))
 	}
 
-	i, err := strconv.ParseInt(c.Args().Get(1), 10, 64)
+	if len(args) > 4 {
```

Comment:
It seems like you're only using two command line arguments: [address, amount]. Why the additional check if `> 4`?
I would also use `c.NArg()` to replace the two `len(c.Args())` check above: https://github.com/urfave/cli/blob/8e01ec4cd3e2d84ab2fe90d8210528ffbb06d8ff/context.go#L182-L185

Reply:
Oops, left over from the custom parsing. Thanks.

Reply:
76ee65f